Dockerfile

```diff
@@ -18,10 +18,15 @@ RUN cat rust-toolchain.toml | grep "channel" | awk '{print $3}' | sed 's/\"//g'
     && cargo install cargo-chef
 
 FROM chef AS planner
+ARG BUILD_DEBUG=false
+ENV CARGO_PROFILE_RELEASE_DEBUG=$build_debug
```

Comment:
Why lowercase `$build_debug` and not `$BUILD_DEBUG`?

Reply:
overlook, thanks for catching that!